build.sbt

```diff
@@ -54,9 +54,9 @@ libraryDependencies ++= Seq(
   "javax.servlet"                   %  "javax.servlet-api"            % "3.1.0"          % "provided",
   "junit"                           %  "junit"                        % "4.12"           % "test",
   "org.scalatra"                    %% "scalatra-scalatest"           % ScalatraVersion  % "test",
-  "org.mockito"                     %  "mockito-core"                 % "2.7.16"         % "test",
+  "org.mockito"                     %  "mockito-core"                 % "2.7.22"         % "test",
   "com.wix"                         %  "wix-embedded-mysql"           % "2.1.4"          % "test",
-  "ru.yandex.qatools.embed"         %  "postgresql-embedded"          % "1.14"           % "test"
+  "ru.yandex.qatools.embed"         %  "postgresql-embedded"          % "1.23"           % "test"
```

Comment:
> Does it work with Java9 if postgresql-embedded is rolled back?

It seems that ```postgresql-embedded``` does not work, nor is supposed to work with Java 9.

It's not even tested with Java 9:
https://github.com/yandex-qatools/postgresql-embedded/blob/master/.travis.yml

I tried to run it with Java 9 but can't get it work so far:
https://travis-ci.org/aadrian/postgresql-embedded/jobs/227742380

The fact that some older version in combination with other older libraries worked with Java 9 is just a pure coincidence IMHO.

I raised an issue https://github.com/yandex-qatools/postgresql-embedded/issues/76 but so far no answer/solution.

Not sure why GitBucket must work with Java 9, when most libraries it depends on, don't even test against it, e.g. it's own libraries like https://github.com/gitbucket/solidbase/blob/master/.travis.yml  :) .